#Discussion

Research on DNA methylation to date has revealed that its extent and function varies considerably among organisms. While several studies have compared DNA methylation patterns between distantly related taxa (\cite{20395474}, \cite{22328716}), our analysis focused on a single taxonomic group, the reef-building scleractinian corals. Within this group, we took a comparative approach of six species to provide a comprehensive evaluation of germline methylation patterns. In three of these species, we found that genes expressed differentially in response to environmental stressors exhibited significantly lower levels of methylation. 

As in most other invertebrate taxa surveyed (\cite{20799955}, \cite{22328716}), we observed patterns of CpG O/E that were indicative of bimodal distributions in all of the coral transcriptomes. All distributions were dominated by a high CpG O/E component, suggesting that the majority of genes in reef corals experience relatively low levels of methylation. A similar pattern was observed in a genome-wide analysis of the sea anemone *Nematostella vectensis* (\cite{20395474}, \cite{22328716}), and in an analysis of exons in *A. millepora* \cite{25511458}. In contrast, CpG O/E profiles in other invertebrates, such as the oyster *Crassostrea gigas* and the sea squirt *Ciona intestinalis*, suggest larger low-CpG O/E components (\cite{20799955}, \cite{22328716}). This is consistent with reports of higher CpG methylation in *C. intestinalis* and *C. gigas* than in *N. vectensis* (21.6%, 15%, and 9.4%, respectively; \cite{20395474}, \cite{24987376}). Levels of DNA methylation broadly reflect evolutionary relationships (\cite{9032274}, \cite{20395474}), so it could be speculated that coral methylation is similar to that of *N. vectensis*. However, phylogenies derived from methylation patterns may differ considerably from those derived from protein sequences, suggesting lineage-specific changes in methylation (\cite{22328716}). A full understanding of the extent of DNA methylation in corals therefore awaits further analysis. 

Across reef coral species, ranking of biological processes according to mean CpG O/E was largely consistent, corroborating a trend reported in several other studies of invertebrate gene body methylation (\cite{20799955}, \cite{22328716}, \cite{25511458}). The biological processes enriched in the high CpG O/E component reflect genes involved in inducible functions, while processes associated with the low CpG O/E component reflect genes for essential housekeeping functions. Housekeeping genes are ubiquitously expressed across tissue types, and tend to be evolutionarily conserved. Indeed, higher levels of germline DNA methylation are associated with gene orthology among invertebrate taxa (\cite{22328716}, \cite{21693438}, \cite{17420183}), a pattern that contrasts with the tendency for methylated cytosines to experience higher mutation rates than nonmethylated nucleotides. It may be that DNA methylation has an overall protective influence despite the mutagenic effect on CpGs, or that CpG mutations have largely run their course over time in heavily methylated genes, with fewer methylated CpGs left for substitutions \cite{21693438}. 

Unlike the repressive effect of gene promoter methylation on gene expression in vertebrates, gene body methylation in invertebrates appears to serve a very different function. While there have been some reports of negative associations between gene body methylation and gene expression in invertebrates (\cite{17420183}, \cite{23877618}), the relationship appears to be largely positive across several taxa, including corals (\cite{20395474}, \cite{24282674}, \cite{25511458}). In *A. millepora*, Dixon et al. \cite{25511458} reported that the most highly expressed genes in a reciprocal transplant experiment were those exhibiting higher levels of methylation. These genes were enriched for housekeeping functions. A similar phenomenon of increased expression among hypermethylated genes was reported for *C. gigas*, in addition to an inverse relationship between DNA methylation and variation in expression between tissues \cite{24282674}. These findings support the hypothesis that gene body methylation reduces transcriptional noise and facilitates consistent expression of ubiquitously expressed core genes needed for essential biological functions (\cite{7732579}, \cite{22232607}, \cite{24397979}). 

In contrast to hypermethylation, sparse methylation of gene bodies, and the tendency for genes involved in inducible functions to have lower methylation levels, has been hypothesized to leave these genes open to a greater variety of transcriptional opportunities \cite{22232607}. Potential sources of transcriptional variation could include access to alternative start sites, increased sequence mutations, exon skipping, and transient methylation \cite{22232607}. Such transcriptional variation could contribute to phenotypic plasticity, and it might be particularly beneficial among species that experience variable environments that require constant tuning of gene expression, such as corals and other sessile organisms. Dixon et al. \cite{25511458} found support for this hypothesis in *A. millepora*, showing that genes differentially expressed in response to transplantation to new environments were significantly enriched in the low methylation (high CpG O/E) component. 

Our analysis of DEGs in response to thermal stress and OA in the three acroporid corals provides further support for the hypothesis that low methylation is associated with transcriptional plasticity. Perhaps unsurprisingly, increased representation of genes involved in inducible functions and reduced representation of housekeeping genes appears to be partially responsible for the lower methylation among DEGs. This was especially true in *A. hyacinthus*. While unsurprising, this highlights an important caveat to our analysis; low methylation among DEGs may not be attributed to the influence of methylation (or lack thereof) per se, but could simply reflect the fact that environmental stressors elicit higher expression of inducible genes relative to housekeeping genes. Furthermore, it remains unclear whether gene body DNA methylation actively regulates genes, as opposed to the alternative hypothesis that it is simply a byproduct of an open chromatin state \cite{22577155}. Future studies will be needed to test for causative relationships between DNA methylation and transcriptional activity. However, experimental studies on social insects have already provided compelling evidence for links between DNA methylation, transcription, and phenotypic plasticity (\cite{18339900}, \cite{19556545}). Studies of DNA methylation in plants have gone even further, having already incorporated epigenetics into ecological research \cite{18021243}.  

Perhaps the most intriguing aspect of DNA methylation is that its effects may extend beyond an individual organismâ€™s lifetime and be transferred to successive generations. Although the epigenomes of some organisms, such as mammals, are extensively reprogrammed during meiosis and embryogenesis, in some cases DNA methylation can be passed on to offspring \cite{24719220}. This is especially true in plants (\cite{21515434}, \cite{24679529}), but there is also evidence for inheritance of DNA methylation in oysters \cite{Olson_2014}. Transgenerational epigenetic inheritance is a potentially transformative biological concept, but its extent and significance is only just beginning to be understood, and remains largely unstudied in the vast majority of organisms (\cite{23416892}, \cite{24679529}). 

Does DNA methylation underlie phenotypic plasticity in corals, acting as a mediator that shapes the phenotype in response to the environment? While we have some evidence that this is indeed the case in some organisms, most of what we know comes from just a few model species. These studies have also revealed that the extent and function of DNA methylation varies considerably among organisms. Thus, only by investigating DNA methylation in a wide range of taxa will we understand its full significance. Reef corals are not only excellent model systems in which to study DNA methylation \cite{25511458}, but whatever we can learn about the basis of their resilience to environmental change may help conserve them. 

    
    
    
    