_**Steven - if you are looking at this I suggest you disregard it for now, it is incomplete and disorganized**_

Research on DNA methylation to date has revealed that its extent and function varies considerably among organisms. While several studies have compared DNA methylation patterns between distantly related taxa (Zemach et al. 2010, Sarda et al. 2012), our analysis focused on a single taxonomic group, the reef-building scleractinian corals. Within this group, we took a comparative approach of six species to provide a comprehensive evaluation of germline methylation patterns.

As in most other invertebrate taxa surveyed to date (Gavery and Roberts 2010, Sarda et al. 2012), we observed bimodal distributions of CpG O/E in all of the coral species. All distributions were dominated by the high-CpG O/E component, suggesting that the majority of genes have relatively low levels of methylation. Similarly, in Nematostella and Acropora millepora 

In contrast to the repressive effect of gene promoter methylation on gene expression in vertebrates, gene body methylation in invertebrates appears to serve a very different function. While there have been some reports of negative associations between gene body methylation and gene expression in invertebrates (Suzuki et al. 2007, Riviere et al. 2013), the relationship appears to be largely positive across several taxa, including corals (Zemach et al. 2010, Gavery and Roberts 2013, Dixon et al. 2014). 

Reef corals are symbiotic organisms that engage in obligate associations with symbiotic dinoflagellates that provide a significant proportion of their nutrition. Flexible partnerships with different symbiont phylotypes broadens the phenotypic range of some coral species, which has been likened to an epigenetic effect (Gilbert et al., 2010). While the influence of symbionts on host thermal tolerance has been a subject of intense research interest, attention has shifted more recently to the role of physiological acclimatization by the host coral. This renewed focus on the host has been spurred in part by the inability of symbiont diversity to fully account for variation in coral thermal tolerance, as well as by the ‘omics’ revolution—the modern, data-heavy approach to molecular biology—that has expanded our ability to understand the molecular basis of organismal physiology (Baird et al., 2009, Evans and Hofmann, 2012). It has long been known that corals are highly phenotypically plastic, but we now are able to investigate the basis of this plasticity. For example, it appears that the heightened thermal tolerance of corals living in warmer environments, such as shallow backreef habitats, can in some cases be explained by higher basal expression levels (“frontloading”) of genes involved in thermal tolerance and stress response (Barshis et al., 2013, Kenkel et al., 2013). Fundamentally, these patterns of gene expression could be under epigenetic control.

A similar manifestation of phenotypic plasticity in corals is the “memory” they are known to develop following exposure to stress or environmental change (Brown et al. 2002). Corals that experience and survive bleaching episodes—the loss of symbionts in response to stress—often develop increased resistance to subsequent bleaching (Maynard et al. 2008). Even within a single coral colony, different sides of the colony can develop a memory of past exposure that may persist for at least a decade (Brown et al. 2014). Based on what is known in other organisms, epigenetic programming is a prime candidate to mediate such memory effects (D’Urso and Brickner, 2014).


One of the primary hypotheses regarding gene body methylation in invertebrates is that it reduces transcriptional noise and facilitates consistent expression of core genes needed for essential biological functions (Bird 1995, Roberts and Gavery 2012, Gavery and Roberts 2014). In contrast, sparse methylation of gene bodies is hypothesized to leave inducible genes open to increased transcriptional opportunities. Potential sources of transcriptional variation could include access to alternative start sites, increased sequence mutations, exon skipping, and transient methylation (Roberts and Gavery 2012). Such transcriptional variation could be the basis of phenotypic plasticity, and it might be particularly beneficial among species that experience variable environments that require constant tuning of gene expression. Future investigations on corals and other species are likely to focus on inducible genes and how their methylation, or lack thereof, influences their expression.


_**Include this paragraph if sequence similarity stuff is used**_
This study provides mild support for the hypothesis that higher levels of germline DNA methylation are associated with evolutionary conservation of gene sequences in reef corals. This association has been reported in several other invertebrate taxa (Sarda et al. 2012, Park et al. 2011, Suzuki et al. 2007), yet the basis of it remains to be understood. The pattern contrasts with the tendency for methylated cytosines to experience higher mutation rates than nonmethylated nucleotides. However, if highly conserved genes are preferentially targeted for DNA methylation, this could have an overall protective influence despite the mutagenic effect on CpGs (Park et al. 2011). Perhaps CpG mutations have already run their course over time in heavily methylated genes, with fewer methylated CpGs left for substitutions (Park et al. 2011). 

_**Possible material for concluding paragraph**_
 Reef corals are not only excellent model systems in which to study DNA methylation, but whatever we can learn about the basis of their resilience to environmental change may help conserve them. In the future, for example, population epigenetics may be considered alongside population genetics to achieve conservation goals.
