#Introduction

As human influence on the planet expands, many organisms must acclimatize and adapt to rapid environmental change. Phenotypic plasticity facilitates a more rapid response to environmental change than is possible through natural selection, and will likely be critical to the persistence of many species \cite{18467590}, \cite{20463950}. Phenotypic change often involves modifications in gene expression. Epigenetic mechanisms, involving alterations to the genome that do not affect the underlying DNA sequence, are increasingly recognized as some of the principal mediators of gene expression \cite{24719220}. The most researched and best understood epigenetic process is DNA methylation, which most commonly involves the addition of a methyl group to a cytosine in a CpG dinucleotide pair. The role of DNA methylation is best understood in mammals, where methylation in promoter regions has a repressive effect on gene expression \cite{11498573}. In plants and invertebrates, methylation of gene bodies prevails, and is thought to be the ancestral pattern \cite{20395474}. Gene body methylation appears to have a range of functions, including regulating alternative splicing, repressing intragenic promoter activity, and reducing the efficiency of transcriptional elongation \cite{24719220}. Methylation of gene bodies may vary according to gene function, and studies on invertebrates indicate that highly conserved genes with housekeeping functions tend to be more heavily methylated than those with inducible functions \cite{22232607}, \cite{22328716}, \cite{25511458}, \cite{24397979}. This has led to speculation that gene body methylation may promote predictable expression of essential genes for basic biological processes, while an absence of methylation could allow for stochastic transcriptional opportunities in genes involved in phenotypic plasticity \cite{22232607}, \cite{25511458}, \cite{24397979}.

Direct relationships between DNA methylation and phenotypic plasticity are increasingly being established. Some examples include caste structure in honeybees and ants \cite{18339900}, \cite{22885060}, expression of the agouti gene in mice (\cite{9707167}), and the influence of prenatal maternal mood on newborn stress levels in humans \cite{18536531}. In many cases, changes in methylation patterns can be attributed to external cues such as temperature, stress, or nutrition. A prime example is the honeybee *Apis mellifera*, where larval consumption of royal jelly induces changes in methylation that ultimately determine the developmental fate of an individual into a queen or a worker \cite{18339900}. Thus, DNA methylation has been established as a key link between environment and phenotype.

Reef-building corals, the organisms that form the trophic and structural foundation of coral reef ecosystems, are known to display a significant degree of phenotypic plasticity \cite{18979594}, \cite{19239678}, \cite{23565725}. As long-lived, sessile organisms, corals are thought to be particularly reliant on phenotypic plasticity to cope with environmental heterogeneity, because they must be able to withstand whatever nature imposes on them over long periods of time \cite{bruno1997clonal}. As phenotypically flexible as they may be, coralsâ€™ longevity and immobility may also contribute to their vulnerability in a changing environment. Reef corals worldwide are experiencing severe declines due to a variety of anthropogenic stressors, including climate change, ocean acidification, and degrading water quality \cite{18079392}. This has raised doubt concerning the ability of corals to survive coming decades. Yet there are also signs that, at least in some cases, corals possess sufficient resiliency to overcome their numerous challenges \cite{24762535}. Recent studies on gene expression variation, for example, support the view that phenotypic plasticity in corals is robust and may provide resilience in the face of ocean warming \cite{23297204}, \cite{23565725}, \cite{24762535}. However, the underlying basis of gene expression variation, and indeed phenotypic plasticity, remain largely unknown.

Evaluation of epigenetic processes therefore represents a logical next step in understanding coral gene expression and phenotypic variation. While recent annotation of the *Acropora digitifera* genome revealed a broad repertoire of genes involved in DNA methylation and other epigenetic processes \cite{23889801}, to date, only one study has investigated possible roles of epigenetic processes in corals \cite{25511458}. Germline DNA methylation patterns in the transcriptome of *Acropora millepora* corroborated findings reported in studies of other invertebrate species \cite{25511458}. Most interestingly, genes that were differentially expressed in response to a common garden transplantation experiment were among the genes exhibiting lower levels of germline methylation \cite{25511458}. This finding further supports studies on invertebrates showing that hypomethylated genes tend to be those with inducible functions \cite{20799955}, \cite{22328716}.

Coral gene expression studies continue to expand, providing rich datasets to further probe the relationship between DNA methylation and gene function. In this study, we performed a comprehensive evaluation of germline methylation patterns in reef corals by examining the transcriptomes of six scleractinian coral species. Germline methylation levels in these data were inferred based on the hypermutability of methylated cytosines, which leads to a reduction in CpG dinucleotides over evolutionary time \cite{2352943}. These data were then matched with gene ontology information, permitting evaluation of methylation patterns associated with broad categories of biological processes. Lastly, in three of the six species, we evaluated germline methylation patterns in genes involved in response to thermal stress and ocean acidification. 
    
    
  
  