#Methods

_**Transcriptome data**_

Germline methylation patterns in the transcriptomes and environmental response genes of six scleractinian coral species were evaluated, including *Acropora hyacinthus*, *A. millepora*, *A. palmata*, *Orbicella faveolata, *Stylophora pistillata*, and *Pocillopora damicornis*. Table 1 (in progress) describes the data sources used in the analyses. 

#####Table 1.Transcriptomes used in the study
Organism | Transcriptome reference | Data link | # Contigs
--------- | ---------- | --------- | --------
*Acropora hyacinthus* | Barshis et al. (2013) | [URL](http://palumbi.stanford.edu/data/33496_Ahyacinthus_CoralContigs.fasta.zip) | 33,496
*Acropora millepora* | Moya et al. (2012) | [URL](http://www.ncbi.nlm.nih.gov/nuccore?term=74409%5BBioProject%5D) | 52,963
*Acropora palmata* | Polato et al. (2011) | [URL](https://usegalaxy.org/datasets/cb51c4a06d7ae94e/display?to_ext=fasta) | 88,020
*Pocillopora damicornis* | Vidal-Dupiol et al. (2013) | [URL](http://2ei.univ-perp.fr/telechargement/transcriptomes/blast2go_fasta_Pdamv2.zip) | 72,890
*Stylophora pistillata* | Karako-Lampert et al. (2014) | [URL](http://data.centrescientifique.mc/Data/454Isotigs.fas.zip) | 15,052

######*Acropora hyacinthus*

The *Acropora hyacinthus* transcriptome was [downloaded](http://palumbi.stanford.edu/data/33496_Ahyacinthus_CoralContigs.fasta.zip) on 1/15/2015. It was featured in an article by [Barshis et al. (2013)](http://www.pnas.org/content/110/4/1387.abstract). The [supporting information file](http://www.pnas.org/content/suppl/2013/01/02/1210224110.DCSupplemental/pnas.201210224SI.pdf) contains detailed information on the experimental conditions and the methods used to create the transcriptome. Briefly, the transcriptome contains 33,496 contigs with high similarity to other cnidarian sequences. The authors used coral and dinoflagellate sequences to filter out dinoflagellate sequences, so the transcriptome contains mostly coral transcripts. 

The same study also evaluated differential gene expression in response to 72 h of thermal stress. A total of 484 contigs were differentially expressed between control and heated corals, regardless of source population. Data on differentially expressed contigs was obtained from a [supplementary information file](http://www.pnas.org/content/suppl/2013/01/02/1210224110.DCSupplemental/sd01.xlsx).

######*Acropora millepora*

The *Acropora millepora* transcriptome was [downloaded](http://www.ncbi.nlm.nih.gov/nuccore?term=74409%5BBioProject%5D) on 2/5/2015. It was featured in an article by [Moya et al. (2012)](http://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05554.x/full). The transcriptome consists of 52,963 contigs and was developed from newly settled polyps without symbionts.

The study above also evaluated differential gene expression in response to ocean acidification. Information on differentially expressed contigs was derived from a [supplementary data file](http://onlinelibrary.wiley.com/store/10.1111/j.1365-294X.2012.05554.x/asset/supinfo/MEC_5554_sm_FigS1-S3_TableS1-S9.pdf?v=1&s=8986ebf969c7552ad857973fff80a0752f87a129).

[Shearer et al. (2014)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0114525#pone.0114525.s002) evaluated differential gene expression of *A. millepora* in response to macroalgal contact via a microarray study. Data on differentially expressed genes in response to macroalgal contact over a period of 6 or 48 hours was derived from list of accession numbers from a [supplementary file](http://journals.plos.org/plosone/article/asset?unique&id=info:doi/10.1371/journal.pone.0114525.s002). These accession numbers were used to retrieve a FASTA file of differentially expressed genes from GenBank.

######*Acropora palmata*

The *Acopora palmata* transcriptome was [downloaded](https://usegalaxy.org/datasets/cb51c4a06d7ae94e/display?to_ext=fasta) on 2/19/2015. It was featured in an article by [Polato et al. 2011](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0028634) and was developed from symbiont-free larval tissue. The transcriptome consists of 88,020 contigs.

A study by [Polato et al. (2013)](http://onlinelibrary.wiley.com.offcampus.lib.washington.edu/doi/10.1111/mec.12163/abstract) evaluated differential gene expression in of *A. palmata* larvae in response to either one or two days of thermal stress. Data on differentially expressed contigs were obtained from a [supplementary file](http://datadryad.org/bitstream/handle/10255/dryad.39350/SuppTableS3_Final.xlsx?sequence=1).

(Note: also cite the following in refs)
Additionally, please cite the Dryad data package:
Polato NR, Baums IB, Altman NS (2013) Data from: Variation in the transcriptional response of threatened coral larvae to elevated temperatures. Dryad Digital Repository. http://dx.doi.org/10.5061/dryad.t3pr6

######*Pocillopora damicornis*

The *Pocillopora damicornis* transcriptome was [downloaded](http://2ei.univ-perp.fr/telechargement/transcriptomes/blast2go_fasta_Pdamv2.zip) on 2/19/2015. It was featured in an article by [Vidal-Dupiol et al. (2013)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0058652). The transcriptome was developed from adult *P. damicornis* and consists of 72,890 contigs. According to the authors, "...27.7% and 69.8% of the contigs were predicted to belong to the symbiont and the host transcriptome, respectively. The remaining contigs could not be taxonomically attributed." 

The study above also evaluated the transcriptomic response of *P. damicornis* to ocean acidification. A list of 48 differentially expressed contigs in response to ocean acidification (including putative dinoflagellate genes) was derived from a [supplementary file](http://journals.plos.org/plosone/article/asset?unique&id=info:doi/10.1371/journal.pone.0058652.s001). The authors developed this relatively short list of contigs based on GO term enrichment analysis of nearly 9,000 contigs (16% of all contigs that were mapped to the reference transcriptome) that were differentially expressed in response to OA conditions. 

A related study by [Vidal-Dupiol et al. (2014)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0107672) evaluated the transcriptomic response of *P. damicornis* to a bacterial pathogen. A list of 44 differentially expressed contigs involved in the innate immune response of *P. damicornis* was derived from a [supplementary file](http://journals.plos.org/plosone/article/asset?unique&id=info:doi/10.1371/journal.pone.0107672.s003). The study used the same reference transcriptome created in [Vidal-Dupiol et al. (2013)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0058652). As in the ocean acidification study, the 44 putative immune response genes were selected from a much larger set of differentially expressed genes using a GO term enrichment analysis. 

######*Stylophora pistillata*

The file Spist.fasta is a *Stylophora pistillata* transcriptome [downloaded](http://data.centrescientifique.mc/Data/454Isotigs.fas.zip) on 2/19/2015. This transcriptome was featured in an article by [Karako-Lampert et al. (2014)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0088615). The transcriptome consists of 15,052 contigs and was developed from adult coral tissue, but the authors determined that only approximately 2.5% of the sequences mapped exclusively to the *Symbiodinium* genome. Thus, the vast majority of transcripts are coral transcripts.


_**Transcriptome annotation**_

All transcriptomes were annotated using Blastx (version 2.2.29) for comparison to the UniProt/Swiss-Prot protein database (version 2/17/2015). Parameters for blastx were as follows: maximum number of target sequences = 1; maximum number of high-scoring pairs = 1; E-value = 10<sup>-5</sup>; number of threads = 8. The output file was uploaded to SQLShare and joined by Swiss-Prot ID to a GOslim annotation table. Only GOslim biological process (’P’) terms were included. This join process allowed an individual contig to fall into more than one GOslim bin, but did not allow it to occur in the same bin more than once. The result was an annotation file including contig IDs and GOslim IDs.

_**Predicted germline methylation**_

Germline methylation levels were inferred based on the hypermutability of methylated cytosines, which tend towards conversion to thymines over evolutionary time. This results in a reduction in CpG dinucleotides, meaning that historically heavily methylated genomic regions are associated with reduced numbers of CpGs. We used an IPython notebook workflow as a pipeline for analysis of CpG O/E. CpG O/E 
was calculated as:

CpG O/E = (number of CpG / number of C x number of G) x (l<sup>2</sup>/l-1)

where l is the number of nucleotides in the contig.

To permit analysis of these data based on GOslim terms, the data were joined with the GOslim annotation file described above based on common contig ID numbers. However, because the GOslim annotation file contained contigs that were duplicated if they fell into more than one GOslim bin, whole-transcriptome CpG O/E patterns were analyzed using the data generated prior to joining with GOslim annotation files.

Analyses were perfomed on a computer running Ubuntu 14.04 with Python 2.7.