\textbf{DNA methylation in reef-building corals: patterns and potential functional roles}

James L. Dimond, Steven B. Roberts

School of Aquatic and Fishery Sciences, University of Washington, Seattle, WA


\textbf{ABSTRACT}

DNA methylation is an epigenetic mark that plays an important yet inadequately understood role in gene regulation, particularly in non-model species. Because it can be influenced by the environment and potentially transferred to subsequent generations, DNA methylation may contribute to the ability of organisms to acclimatize and adapt to environmental change. Here we will evaluate the distribution of gene body methylation in reef-building corals, a group of organisms facing a number of significant environmental challenges. Gene body methylation in six species of corals will be inferred from in silico transcriptome analysis of CpGo/e ratios, which are highly correlated with patterns of methylation enrichment. Consistent with what has been documented in other invertebrates, preliminary results suggest that hypermethylated genes are those with housekeeping functions, while hypomethylated genes tend to have inducible functions. We will therefore also test the hypothesis that genes that are differentially expressed in response to environmental change tend to be sparsely methylated. This may yield insight into the potential role of methylation in mediating phenotypic plasticity and acclimatization. 

\textbf{Introduction}
 
The capacity of a genotype to produce a range of phenotypes in response to environmental heterogeneity is known as phenotypic plasticity. Environmental variability is thought to drive the evolution of phenotypic plasticity, as a greater phenotypic range will promote success in variable or changing environments. Because phenotypic plasticity permits a more rapid response to environmental change than is possible through adaptation, it may become especially critical to the persistence of many species as anthropogenic global change expands and intensifies (Charmantier et al., 2008, Chevin et al., 2010). 
 
Phenotypic change often involves modifications in gene expression. Epigenetic mechanisms, involving alterations to the genome that do not affect the underlying DNA sequence, are increasingly recognized as some of the principal mediators of gene expression (Duncan et al., 2014). The most researched and best understood epigenetic process is DNA methylation, which most commonly involves the addition of a methyl group to a cytosine in a CpG dinucleotide pair. The role of DNA methylation is best understood in mammals, where methylation in promoter regions has a repressive effect on gene expression (Jones and Takai, 2001). In plants and invertebrates, methylation of gene bodies prevails, and is thought to be the ancestral pattern (Zemach et al., 2010). Gene body methylation appears to have a range of functions, including regulating alternative splicing, repressing intragenic promoter activity, and reducing the efficiency of transcriptional elongation (Duncan et al., 2014). Methylation of gene bodies may vary according to gene function, and studies on invertebrates indicate that genes with housekeeping functions tend to be more heavily methylated than those with inducible functions (Roberts and Gavery, 2011, Sarda et al., 2012, Dixon et al., 2014, Gavery and Roberts, 2014). This has led to speculation that gene body methylation may promote predictable expression of essential genes for basic biological processes, while an absence of methylation could allow for stochastic transcriptional opportunities in genes involved in phenotypic plasticity (Roberts and Gavery 2012, Dixon et al., 2014, Gavery and Roberts, 2014).
 
Direct relationships between DNA methylation and phenotypic plasticity have only recently been established. Some examples include caste structure in honeybees and ants (Kucharski et al. 2008, Bonasio et al. 2012), expression of the agouti gene in mice (Wolff et al. 1998), and the influence of prenatal maternal mood on newborn stress levels in humans (Oberlander et al. 2008). In many cases, changes in methylation patterns can be attributed to external cues such as temperature, stress, or nutrition. A prime example is the honeybee Apis mellifera, where larval consumption of royal jelly induces changes in methylation that ultimately determine the developmental fate of an individual into a queen or a worker (Kucharski et al. 2008). Thus, DNA methylation has been established as a key link between environment and phenotype.

Reef-building corals, the organisms that form the trophic and structural foundation of coral reef ecosystems, are among the numerous organisms about which we know very little regarding their epigenetics. However, it is known that corals display a significant degree of phenotypic plasticity (Todd 2008, Granados-Cifuentes et al., 2013). As long-lived, sessile organisms, corals are thought to be particularly reliant on phenotypic plasticity to cope with environmental heterogeneity, because they must be able to withstand whatever nature imposes on them over long periods of time (Bruno and Edmunds, 1997). As phenotypically flexible as they may be, coralsâ€™ longevity and immobility may also contribute to their vulnerability in a changing environment. Reef corals worldwide are experiencing severe declines due to a variety of anthropogenic stressors, including climate change, ocean acidification, and degrading water quality (Hoegh-Guldberg et al., 2007). This has raised doubt concerning the ability of corals to survive coming decades. Yet there are also signs that, at least in some cases, corals possess sufficient resiliency to overcome their numerous challenges (Palumbi et al., 2014). Recent studies on gene expression variation, for example, support the view that phenotypic plasticity in corals is robust and may provide resilience in the face of ocean warming (Barshis et al., 2013, Granados-Cifuentes et al., 2013, Palumbi et al., 2014). However, the underlying basis of gene expression variation, and indeed phenotypic plasticity, remain largely unknown.

Evaluation of epigenetic processes therefore represents a logical next step in understanding coral gene expression and phenotypic variation. To date, only one study has investigated epigenetic processes in corals (Dixon et al., 2014). Germline DNA methylation patterns in the transcriptome of Acropora millepora corroborated findings reported in studies of other invertebrate species (Dixon et al., 2014). Most interestingly, genes that were differentially expressed in response to a common garden transplantation experiment were among the genes exhibiting lower levels of germline methylation (Dixon et al., 2014). This finding provides more direct support for earlier studies on invertebrates showing that hypomethylated genes tend to be those with inducible functions based on gene ontology (GO) information (Gavery and Roberts, 2010, Sarda et al., 2012).

The link between inducible gene expression and low germline methylation can be probed even further using the abundant data on coral gene expression in response to elevated temperature, ocean acidification, and other stressors (e.g. Meyer et al., 2011, Moya et al., 2012). In this study, we evaluate germline methylation patterns in the transcriptomes and environmental response genes of six scleractinian coral species. Germline methylation levels in these data are inferred based on the hypermutability of methylated cytosines, which leads to a reduction in CpG dinucleotides over evolutionary time (Sved and Bird 1990). These data are then matched with gene ontology information, permitting evaluation of methylation patterns associated with broad categories of biological processes. 


\textbf{Methods}

Germline methylation patterns in the transcriptomes and environmental response genes of six scleractinian coral species were evaluated, including Acropora hyacinthus, A. millepora, A. palmata, Montastraea faveolata, Stylophora pistillata, and Pocillopora damicornis. Table 1 (in progress) shows the datasets used in the analyses. Germline methylation levels were inferred based on the hypermutability of methylated cytosines, which tend towards conversion to thymines over evolutionary time. This results in a reduction in CpG dinucleotides, meaning that historically heavily methylated genomic regions are associated with reduced numbers of CpGs. We developed an iPython script that calculated CpG ratio (also known as CpG O/E) as:

\begin{quote}
CpG O/E = (number of CpG / number of C x number of G) x (l^2/l-1)
\end{quote}

where l is the number of nucleotides in the contig.

Next, we developed an iPython script that performed a blastx query of the transcriptome datasets against the SWISS-PROT database, returning the best predicted protein match with an e-value of \begin{quote}
>= 10^-5
\end{quote}. SWISS-PROT IDs were then joined with biological processes GO Slim terms from [which database?] using SQLite3. An individual contig may have fallen into more than one GO Slim bin, but was not permitted to occur in the same bin more than once.
